be/src/storage/rowset/rowset.cpp

```diff
@@ -87,4 +87,11 @@ void Rowset::make_commit(int64_t version, uint32_t rowset_seg_id) {
     make_visible_extra(v);
 }
 
+bool Rowset::has_txn_meta() {
+    if (_rowset_meta->get_meta_pb().has_txn_meta()) {
+        return true;
+    }
+    return false;
```

Comment:
the logic is obviously, can return directly, since the has_txn_meta() is already a boolean value.
```suggestion
    return _rowset_meta->get_meta_pb().has_txn_meta();
```